MusicDB/3 reports.sql

```diff
@@ -0,0 +1,30 @@
+
+--Reports:
+--1) I need to know if I have any repeat customers and what they rented
```

Comment:
-- LL the select and group by should include the customer first name as well (2 customers can have the same last name)